recipes/postgresql/bld.bat

```diff
@@ -0,0 +1,28 @@
+cd src\tools\msvc
```

Comment:
The first thing I will do once this s merged is to open a PR for the Windows build. So my lazy side says it should stay :smile: